css/style.css

```diff
@@ -533,6 +533,7 @@ figcaption details p {
     text-decoration: none;
     border-bottom: none;
     padding: 0 0.2em;
+    font-size:.84em;
```

Comment:
The links still overlap. Will you reduce the font-size even more to avoid it?
![Screenshot from 2020-03-02 21-08-06](https://user-images.githubusercontent.com/34926285/75691245-011c4300-5cca-11ea-84e6-007640c31175.png)

Reply:
Its up to the user whether he/she wants to decrease that much line space or not!